src/components/ConversationWindow/ConversationWindow.jsx

```diff
@@ -44,7 +44,7 @@ ConversationWindow.propTypes = {
       lname: PropTypes.string,
     }).isRequired,
   }).isRequired,
-  toneResult: PropTypes.array.isRequired,
+  toneResult: PropTypes.object.isRequired,
```

Comment:
Since you are accessing two properties of toneResult, let's add it to the validation IE:
```
PropTypes.shape({
  toneSummary: PropTypes.object.isRequired,
  toneHistory: PropTypes.arrayOf(PropTypes.object).isRequired),
})

```

Don't need to validate further since we will validate the object itself inside of the ToneBox / ToneHistory. Just mostly want to validate types for anything we access.